src/Pos/Update/Poll/Logic/Apply.hs

```diff
@@ -294,9 +294,7 @@ applyDepthCheck hh cd
     | otherwise = do
         deepProposals <- getDeepProposals (cd - blkSecurityParam)
         let winners =
-                concatMap toList $
-                map resetAllDecisions $
-                map (NE.sortBy proposalCmp) $
+                concatMap ((toList . resetAllDecisions) . NE.sortBy proposalCmp) $
```

Comment:
😎
Makes sense. I'll save all these redundant brackets for the next winter, then.